File: backend/src/main/kotlin/com/example/backend2/global/messaging/dto/MessagePayload.java
package com.example.backend2.global.messaging.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MessagePayload {
    private String eventType;  // 이벤트 타입 (예: AUCTION_BID, USER_SIGNUP 등)
    private String sender;      // 발신자 정보
    private Object data;       // JSON 형태 데이터
}

File: backend/src/main/kotlin/com/example/backend2/global/messaging/dto/MessagePayload.kt
package com.example.backend2.global.messaging.dto

class MessagePayload {
    val eventType: String? = null // 이벤트 타입 (예: AUCTION_BID, USER_SIGNUP 등)
    val sender: String? = null // 발신자 정보
    val data: Any? = null // JSON 형태 데이터
}
